vendor/engines/split_accounts/app/models/split_accounts/reports/export_raw_transformer.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-require "hash_helper"
+require Rails.root.join("app/lib/hash_helper")
```

Comment:
this can be left as before, also in `spec/teaspoon_env.rb` and `config/initializers/01_nucore.rb`